Learning/data_collection.py

```diff
@@ -55,7 +79,7 @@ def evaluate_policy(self, env, policy, collection_time_steps, sector_rows, secto
     # cpu_cores =  [i for i in range(30, 61)]
     # os.system("taskset -pc {} {}".format(",".join(str(i) for i in cpu_cores), os.getpid()))
     rows = 150
-    cols = 300
+    cols = 150
```

Comment:
Let's have all rows and cols variables reference the ROWS and COLS in sim_globals.py.  Same for the other vars below that were suggested in collect.py.